content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -942,3 +942,126 @@ a scheduled action, but you can also send each invoice manually if needed.
 
 Once the invoice is validated with |SUNAT|, customers can download the :file:`.zip` with the CDR,
 XML, and PDF files directly from the customer portal by clicking the :guilabel:`Download` button.
+
+Reports
+=======
+
+Permanent Inventory Reports: PLE 12.1 and PLE 13.1
+--------------------------------------------------
```

Comment:
Please use [sentence case for headings](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#document-structure) (and add a custom anchor for later use) :)
```suggestion
.. _peru/reports-ple:

Permanent inventory reports: |PLE| 12.1 and |PLE| 13.1
------------------------------------------------------
```